src/main/java/pl/noip/lolstats/lol/stats/dto/Repository.java

```diff
@@ -0,0 +1,13 @@
+package pl.noip.lolstats.lol.stats.dto;
+
+import java.util.List;
+
+import org.springframework.data.mongodb.repository.MongoRepository;
+import pl.noip.lolstats.lol.stats.model.Account;
+
+public interface Repository extends MongoRepository<Account, String> {
+
+     Account findByEmail(String email);
```

Comment:
since email is id, method for finding account by email already exists and it's findOne